docs/tagging.md

```diff
@@ -96,6 +96,80 @@ backup=true
 billingID=ABCDEF
 ```
 
+# Snapshot Tagging
```

Comment:
Do we want this entire section repeated? I think we can just mention that this feature is also available in SnapshotClass with the specified parameter names (and we --extra-create-metadata to enable it).

Reply:
We at least should present some``VolumeSnapshotClass`` examples.

Reply:
I can make this "Snapshot Tagging" section (slightly) more compact, but I strongly believe that we should keep the ``VolumeSnapshotClass`` examples.

Reply:
@rdpsin @ConnorJC3 I just trimmed down this section a little bit, only keeping the essentials. Let me know what you think

Reply:
Looks fine, although I would just keep the last example in.

Reply:
Consolidated the examples and updated the doc